smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/ShapeExt.kt

```diff
@@ -50,7 +48,7 @@ inline fun <reified T : Trait> Shape.hasTrait(): Boolean = hasTrait(T::class.jav
 
 inline fun <reified T : Trait> Shape.expectTrait(): T = expectTrait(T::class.java)
 
-inline fun <reified T : Trait> Shape.getTrait(): T? = getTrait(T::class.java).getOrNull()
+inline fun <reified T : Trait> Shape.getTrait(): T? = getTrait(T::class.java).orElse(null)
```

Comment:
For all of the following codegen test files, there are substantial changes (reductions, really) in generated code.  You can review if you like, but codegen tests were basically only modified with new expected code.